README.md

```diff
@@ -0,0 +1,122 @@
+
+### Mollify CLI Package
+
+Mollify CLI is a command-line tool designed to manage the filesystem for the Mollify Learning Management System (LMS). It ensures that the file structure adheres to SvelteKit's file path conventions and provides a convenient way to create, archive, and remove various entities within the LMS. Mollify CLI is written in TypeScript and uses the Commander and Enquirer libraries to provide a user-friendly and intuitive interface.
+
+### Mollify Molly Package
+
+Mollify Molly is an OpenAI-powered teaching assistant, designed to answer student queries by accessing the lesson content.
+
+### Mollify TTS Package
+
+Mollify TTS provides audio for markdown content, enabling users to generate and listen to the content. Currently leveraging ElvenLabs for TTS, future plans may include various optional TTS packages to offer a more flexible user experience.
+
+### Mollify Flow Package
+
+Mollify Flow offers a user interface for the CLI package, enabling users to create, update, and manage content. It includes a WYSIWYG editor for real-time content preview and update tracking.
+
+### Mollify Types Package
+
+Mollify Types package is home to all TypeScript types and definitions for easy import into any of the Mollify packages.
+
+## Stack
+
+These are a some of the core dependencies although many more are present.
+
+- Lerna
+- Sveltekit
+- TypeScript
+- Tailwind
+- Skeleton UI
+- Vite
+
+## Getting Started
+
+### Installing
+
+1. Clone the repo:
+
+```bash
+git clone https://github.com/Fermain/-mollify.git
+```
+
+2. Install the dependencies:
+
+```
+npm install
+npx lerna bootstrap
+```
+
+3. Create a .env in the LMS package root
+
+```
+OPENAI_API_KEY=YOUR_KEY
+OPENAI_TOKEN_LIMIT=1000
+ELEVENLABS_API_KEY=YOUR_KEY
+```
+
+### Usage
+
+To run the LMS you can use this command from the root, alternatively you can run this on the Molly and Flow branches.
+
+```bash
+npx lerna run dev --scope @mollify/lms
+```
+
+### Future Implementation
+
+The aim in the future will be for a users to simple run to install mollify and its dependencies.
+
+```
+npm i @mollify/LMS
+```
+
+Then create a content folder with markdown using the Flow editor or migrate existing content using
+
+```
+npx mollify migrate
+```
+
+Then a user will be prompted to answer questions relating to the front matter of the content so it can be indexed and run as a website.
+
+## Contributing
+
+Create a fork, find an issue to work on and submit a pull request with your contribution.
+
+## Contact
+
+[![Fermain God Emperor](readme/contact.jpg)](https://github.com/Fermain)
```

Comment:
I think there are a couple of typos here too, but I love the pic!

Reply:
I should have fixed my typo and my constant use of `are` rather than `our`, I know its wrong, I've been told a million times but I can't seem to stop myself.